test/metabase/search/api_test.clj

```diff
@@ -1778,7 +1778,7 @@
                     first
                     :result_metadata))))))))
 
-(deftest prometheus-response-metrics-test
+(deftest ^:synchronized prometheus-response-metrics-test
```

Comment:
What are the synchronized annotations doing?

Reply:
Not completely sure...

The `multiple-limits` test kept failing for me because the index did not exist when it tried to call the search API. It was odd because it uses the same pattern as many other tests here that were passing.

One attempt I had was to add the `^:synchronized` tag I saw on other tests, even though I couldn't find what it actually did off hand. And added it to other that might also be messing with the index to see if that helped. 

It didn't help--it took adding the call to the force-reindex API to get the test to pass. But, I left the synchronzed in since it followed the pattern of other APIs and I I didn't want to mess with a passing CI build :) 

I'm cleaning up this whole test in a separate PR so figured I'd take a closer look at synchroized as part of that.